monitoring-guidelines.md

```diff
@@ -55,13 +55,14 @@ When creating a KubeVirt alert rule, please follow the [OpenShift Alerting Consi
 In addition to the [OpeShift Style Guide](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#style-guide) the KubeVirt alerts MUST include:
 1. `kubernetes_operator_part_of` label indicating the operator name. Value should be set to `kubevirt`.
 2. `kubernetes_operator_component` label indicating the value of the sub operator name.
+3. `operator_health_impact` label indicating how the alert impacts the operator health.
+   Valid values are: `none`, `warning`, or `error`.
```

Comment:
Please elaborate how this label differs from the Severity label and what the different values mean. Also state how it can be used (To filter alerts that compromise the operator health). Also explain why we use the "none" value(To be able to do automatic testing so that developers will not forget to add the label when its needed).

Reply:
Updated it, please let me know if we still miss something.
One thing that I'm not sure of:
Should we explain under this guidelines why we kept the none values instead of not using this label for these alerts? 
IMO, the guidelines should only guide how to implement the approach that was chosen, and not justify why it was chosen.